requirements/connectors/github.in

```diff
@@ -1,5 +1,4 @@
 -c ../common/constraints.txt
 
-# NOTE - pygithub==1.58.0 fails due to https://github.com/PyGithub/PyGithub/issues/2436
-pygithub>1.58.0
+pygithub>=2.4.0
```

Comment:
As mentioned in the call, please also create the `github.txt` with `uv` and python 3.9 (the lowest python version we support). You can install `uv` on osx with homebrew.